LayoutTests/http/tests/dom/focus-in-cross-origin-iframe.html

```diff
@@ -0,0 +1,33 @@
+<script>
+
+if (window.testRunner) {
+    testRunner.dumpAsText();
+    testRunner.waitUntilDone();
+}
+
+window.addEventListener("message", (event) => {
+	document.getElementById("result").innerText = event.data;
+	if (window.testRunner)
+		testRunner.notifyDone()
+})
+
+function startTest()
+{
+	if (!eventSender)
+		return
+		
+	eventSender.keyDown("\t")
```

Comment:
Can we use `UIHelper.keyDown` instead? That'll make this test also work in iOS.

Reply:
Yup, iOS focus is still different enough for it to need an exception here. That's fine.